### ISSUE TITLE
Page loading indicator for docs

### ISSUE THREAD

**Issue by iAdramelk** [iterative/dvc.org#589]
Moved to separate issue from #286.

Right now there is no loading indicator on navigation in docs pages. On slow connection after you click on link there will be a pause for couple of seconds and only after that page will change. Maybe we can show some progress indicator instead so user can understand that everything is working as intended.
